tool/grind.dart

```diff
@@ -273,3 +272,72 @@ ArchiveFile _fileFromString(String path, String contents,
 ArchiveFile _file(String target, String source, {bool executable: false}) =>
     _fileFromBytes(target, new File(source).readAsBytesSync(),
         executable: executable);
+
+/// Generate a SHA256 hash value using the contents of a given [file]
+String _sha256(String file) =>
+    sha256.convert(new File(file).readAsBytesSync()).toString();
+
+@Task('Build Homebrew formulae.')
+//@Depends(package)
+homebrew_formulae() {
+  var hash_x64 = _sha256('build/dart-sass-$_version-macos-x64.tar.gz');
+  var hash_ia32 = _sha256('build/dart-sass-$_version-macos-ia32.tar.gz');
+  var formulae = """
+require 'formula'
+
+class DartSass < Formula
+  homepage 'https://github.com/sass/dart-sass'
+
+  version '$_version'
+  if MacOS.prefer_64_bit?
+    url 'https://github.com/sass/dart-sass/releases/download/$_version/dart-sass-$_version-macos-x64.tar.gz'
+    sha256 '$hash_x64'
+  else
+    url 'https://github.com/sass/dart-sass/releases/download/$_version/dart-sass-$_version-macos-ia32.tar.gz'
+    sha256 '$hash_ia32'
+  end
+
+  def install
+    bin.install "dart-sass"
+    bin.install Dir["src"]
```

Comment:
Homebrew installs the package into ```/usr/local/Cellar/dart-sass/<version>``` directory.

After installation, a symbolic link ```/usr/local/bin/dart-sass``` that's pointing on Cellar place is created.

Only ```/usr/local/bin/dart-sass``` is on the path of the end user.

```
/usr/local/bin/dart-sass -> ../Cellar/dart-sass/1.0.0-alpha.8/bin/dart-sass
```

```
$ ls -1R /usr/local/Cellar/dart-sass/
1.0.0-alpha.8

/usr/local/Cellar/dart-sass//1.0.0-alpha.8:
INSTALL_RECEIPT.json
bin

/usr/local/Cellar/dart-sass//1.0.0-alpha.8/bin:
dart-sass
src

/usr/local/Cellar/dart-sass//1.0.0-alpha.8/bin/src:
DART_LICENSE
SASS_LICENSE
dart
sass.dart.snapshot```